example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,40 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(collection);
+    }
+  });
+
+  it('updating a collection publishes a record to the collection reporting SNS topic', async () => {
+    if (beforeAllFailed) {
+      fail('beforeAll() failed');
+    } else {
+      await apiTestUtils.updateCollection({
+        prefix: config.stackName,
+        collection,
+        updateParams: { files: [] },
```

Comment:
I'm pretty surprised that this update works. I think `collection.files` are required by the collection write code:

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/models/collections.js#L60

So I'm a bit confused how this is passing if you can shed any light

Reply:
Hm interesting. If it were required, then wouldn't the file value here be invalid? https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/lib/testUtils.js#L257 

```
  collection.files.forEach((file) => checkRegex(file.regex, file.sampleFileName));

``` wouldn't this just be undefined if file were `[]`?